Emergence/Contexts/App Setup/AppViewController.swift

```diff
@@ -27,7 +27,9 @@ class AppViewController: UINavigationController {
                 defaults.synchronize()
 
                 self.context.network.authToken = XAppToken(defaults: defaults)
-                completion()
```

Comment:
This is on our library? We could open an issue to fix it there, since I imagine this is a common case (API call, then update UI).